uber/utils.py

```diff
@@ -99,37 +127,36 @@ def completed(self, attendee):
         return matches[0] if matches else None
 
 
```

Comment:
This is one of the holdouts which I haven't yet refactored into a "don't use config we expect to change at the import level" approach.  So the server still definitely needs to be restarted for changes to this, at the moment.